docs/includes/register-sso-add-in-aad-v2-include.md

```diff
@@ -67,28 +129,69 @@ Sometimes called an _application password_, a client secret is a string value yo
     > - `93d53678-613d-4013-afc1-62e9e444a0a5` (Office on the web)
     > - `bc59ab01-8403-45c6-8796-ac3ef710b3e3` (Outlook on the web)
 
-1. Select **Add a client application**. In the panel that opens, set the **Client ID** to the respective GUID and check the box for `api://<fully-qualified-domain-name>/<application-id>/access_as_user`.
+1. Select the checkbox for `api://localhost:44355/<app-id>/access_as_user` in **Authorized scopes**.
 
 1. Select **Add application**.
 
+    :::image type="content" source="../images/azure-portal-add-application.png" alt-text="The Add a client application pane.":::
```

Comment:
![azure-portal-add-application](https://user-images.githubusercontent.com/30682803/194392185-3dc2cd5e-f271-420f-a517-2dde30868488.png)


I cut out the extra white space. Please use this version instead!

Reply:
Done.